### PR TITLE
Adds a new HoS coat

### PR DESCRIPTION
**Pull request by Blithering**
Adds a new design of HoS coat, because Sundowner is the smug coat wearing bastard every HoS wishes to be.
sundownercoat2
screencaphos

The coat spawns in HoS lockers, replacing the duplicate Jensen coat.
plsnoconflicts

### CODE REVIEW

**Comment by kol1th** [comment]
in game screenshots when

**Comment by 9600bauds** [comment]
ingame screenshot

ef8

**Comment by kol1th** [comment]
yes very good

**Comment by yclatious** [comment]
>Traitor HoS going full sundowner on greyshits with energy machetes

Its the kind of thing you never knew you wanted but need.

**Comment by jknpj** [comment]
Kids are cruel clown. And i am very in touch with my inner child!

**Comment by Probe1** [comment]
holy shit 
download 1

**Comment by ghost** [comment]
looks kinda shitty tbh

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -111,7 +111,7 @@
 		new /obj/item/clothing/accessory/holster/knife/boot/preloaded/tactical(src)
 		new /obj/item/clothing/suit/armor/vest(src)
 		new /obj/item/clothing/under/rank/head_of_security/jensen(src)
-		new /obj/item/clothing/suit/armor/hos/jensen(src)
+		new /obj/item/clothing/suit/armor/hos/sundowner(src)
```
Comment (Probe1) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Keep the jensen coat and add the sundowner as well.

**Review by Probe1 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 1 above)
Comment (Blithering) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: I considered that, but thought having an additional HoS armour to hand out every round might upset the balance. I could set it to a 50/50 chance which one spawns maybe?

**Comment by Probe1** [comment]
oh boy dmi conflicts..  Grab the new DMI files from the repo, pop your sprites in and push them.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 1 above)
Comment (Probe1) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: That's acceptable.

**Review by Probe1 — APPROVED** [review]

**Comment by Blithering** [comment]
Fixed some of the janky shading on the sleeves and collar. 
sundownercoat2
Minor change but it would have bugged me if I didn't.

**Pull request merged**